gatsby_fp/src/components/layout.js

```diff
@@ -1,36 +1,47 @@
-/**
- * Layout component that queries for data
- * with Gatsby's useStaticQuery component
- *
- * See: https://www.gatsbyjs.org/docs/use-static-query/
- */
-
 import React from "react"
-import PropTypes from "prop-types"
-import { useStaticQuery, graphql } from "gatsby"
-
+import { graphql } from "gatsby"
 import Header from "./header"
-//import "./layout.css"
+import Fields from "./fields"
+import Helmet from "react-helmet"
 
-const Layout = ({ children }) => {
-  const data = useStaticQuery(graphql`
-    query SiteTitleQuery {
-      site {
-        siteMetadata {
-          title
+export default function Layout({ children, data }) {
+  const sanitizeHtml = require("sanitize-html-react")
+  function sanitizeMarkup(child) {
+    return sanitizeHtml(child, {
+      allowedTags: ["p", "b", "i", "u", "sup", "sub", "br"],
+    })
+  }
+  if (data) {
+    var sanitizedData = data.multiApiSourcePeopleFaculty
+  }
+  function traverse(parent) {
+    for (let child in parent) {
+      if (parent[child] !== null && typeof parent[child] == "object") {
+        //going one step down in the object tree!!
+        traverse(parent[child])
+      } else {
+        if (parent[child]) {
+          parent[child] = sanitizeMarkup(parent[child])
         }
       }
     }
-  `)
+  }
+
+  traverse(sanitizedData)
 
   return (
     <>
       <a href="#page" id="skip_to_content" className="offscreen">
         Skip to Main Content
       </a>
+      <Helmet
```

Comment:
Instead of a void tag, we can add our `<title>...</title>` inside of our `<Helmet>...</Helmet>`. Also, let's discuss other options of where this `<Helmet>` could move to. I think it doesn't really affect the final results but its purpose is to affect our `<head>` contents and `<body>` attributes, which logically do not come between our "Skip to Main Content" link and our main content.